archetype/config/env/production.js

```diff
@@ -1,37 +1,3 @@
-'use strict'
-
-const winston = require('winston')
-
 module.exports = {
 
-  trailpack: {
```

Comment:
@jaumard `disabled` option not working - so it's useless.
Same with logger. Mostly everybody deploy app into heroku. and File logger also useless in there.